Havit/Havit/Screens/AddContents/AddContentViewController.swift

```diff
@@ -155,21 +155,17 @@ final class AddContentViewController: BaseViewController {
 
 extension AddContentViewController: UITextViewDelegate {
     func textViewDidChange(_ textView: UITextView) {
+        let size = CGSize(width: view.frame.width, height: .infinity)
+        let estimatedSize = urlTextView.sizeThatFits(size)
 
-        let fixedWidth = urlTextView.frame.size.width
-        urlTextView.sizeThatFits(CGSize(width: fixedWidth, height: CGFloat.greatestFiniteMagnitude))
-        let newSize = urlTextView.sizeThatFits(CGSize(width: fixedWidth, height: CGFloat.greatestFiniteMagnitude))
-        var newFrame = urlTextView.frame
-        newFrame.size = CGSize(width: max(newSize.width, fixedWidth), height: newSize.height)
-        urlTextView.frame = newFrame
-
-        urlTextView.translatesAutoresizingMaskIntoConstraints = true
-
-        if urlTextView.frame.size.height >= textViewMaxHeight {
-            urlTextView.isScrollEnabled = true
-            urlTextView.frame.size.height = self.textViewMaxHeight
-        } else {
-            urlTextView.isScrollEnabled = false
+        urlTextView.constraints.forEach { (constraint) in
+            if estimatedSize.height >= 125 {
+                urlTextView.isScrollEnabled = true
+            } else {
+                if constraint.firstAttribute == .height {
+                    constraint.constant = estimatedSize.height
+                }
+            }
```

Comment:
p1;

```suggestion
            if estimatedSize.height < 125, constraint.firstAttribute == .height {
                constraint.constant = estimatedSize.height
            } else {
                 urlTextView.isScrollEnabled = true
            }
```
이렇게 쓸 수 있고 각 조건에 대한 변수가 있으면 더 좋겠네요!

Reply:
@YoonAh-dev 저 혹시 ~ 이렇게 하면 두근 거림 현상이 다시 살아나느데 왜 그럴까요 ?? 의문선배

Reply:
@YoonAh-dev  계세요~~? 계신가요 ~~>?

Reply:
@noah0316  어풀읍 ! 어풀읍 !

Reply:
@YoonAh-dev  스크롤을 먼저 막아야 두근거림을 해결할 수있어서 스크롤 먼저 분기처리를 해야하는 것 같습니다

Reply:
@YoonAh-dev 나를 봐달라 ! 나를 봐달라 ! 둑흔 둑흔

Reply:
@YoonAh-dev  날봐 날봐 귀순 ~ 날봐나로바 귀순 ~